tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -253,7 +255,10 @@ def _gen_kernel_library(
         unroll_factors_override = {},
         extra_args = [],
         test_tags = [],
-        test_size = "medium"):
+        test_size = "medium",
+        jit_i64_indexed_for_large_tensors_types = None,
+        output_jit_i64_indexed_for_large_tensors_types = None
```

Comment:
Should these (`jit_i64_indexed_for_large_tensors_types` and `output_jit_i64_indexed_for_large_tensors_types`) be initialized to `[]`?

Reply:
changed.